dolphinscheduler-task-plugin/dolphinscheduler-task-pigeon/src/test/java/org/apache/dolphinscheduler/plugin/task/pigeon/PigeonTaskTest.java

```diff
@@ -67,7 +68,7 @@ public void before() throws Exception {
         // "\",\"type\":\"VARCHAR\",\"value\":\"127.0.0.1:8080\"}]");
         Map<String, String> gloabParams =
                 Collections.singletonMap(PigeonTask.KEY_POOL_VAR_PIGEON_HOST, "127.0.0.1:8080");
-        Mockito.when(taskExecutionContext.getDefinedParams()).thenReturn(gloabParams);
+        Mockito.when(ParameterUtils.convert(taskExecutionContext.getPrepareParamsMap())).thenReturn(gloabParams);
```

Comment:
please fix the CI error

<img width="1219" alt="image" src="https://github.com/apache/dolphinscheduler/assets/38122586/e9a6721a-fb31-4526-949f-c253a078d7e0">

Reply:
Modified Complete